src/main/java/seedu/letsgethired/logic/parser/ArgumentMultimap.java

```diff
@@ -75,4 +75,13 @@ public void verifyNoDuplicatePrefixesFor(Prefix... prefixes) throws ParseExcepti
             throw new ParseException(Messages.getErrorMessageForDuplicatePrefixes(duplicatedPrefixes));
         }
     }
+
+    /**
+     * Throws a {@code ParseException} if more than one prefix is found in the multimap
+     */
+    public void verifyOnlyOnePrefix() throws ParseException {
+        if (argMultimap.size() > 2) {
```

Comment:
Wait why is the size 2 instead of 1 here? Since we are checking if there is only one prefix?

Reply:
There is the prefix preamble, and the first prefix after the preamble. So in total, there are 2 elements in the multimap